src/diffusers/models/unet_2d_condition.py

```diff
@@ -98,6 +98,7 @@ def __init__(
             "DownBlock2D",
         ),
         up_block_types: Tuple[str] = ("UpBlock2D", "CrossAttnUpBlock2D", "CrossAttnUpBlock2D", "CrossAttnUpBlock2D"),
+        do_self_attention: Tuple[bool] = (True, True, True, True),
```

Comment:
only some layers use "double cross attention"?

Reply:
Actually ok for me as is - don't think we have to add new blocks for this.
Just in terms of naming I'd call it "double_cross_attention"

Reply:
```suggestion
        double_cross_attention: Union[bool, Tuple[bool]] = False,
```

Reply:
As @pcuenca mentioned, `disable_self_attention: Union[bool, Tuple[bool]] = False` might be good as well. But double negatives are always a bit hard to grasp. 

Maybe:
`only_cross_attention` could also be a good config name. @pcuenca @anton-l @patil-suraj wdyt?

Reply:
I like `only_cross_attention` !